contracts/instruments/RibbonThetaVault.sol

```diff
@@ -281,22 +297,53 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @param share is the number of vault shares to be burned
      */
     function withdraw(uint256 share) external nonReentrant {
-        uint256 withdrawAmount = _withdraw(share, false);
+        uint256 withdrawAmount = _withdraw(share, false, false);
         IERC20(asset).safeTransfer(msg.sender, withdrawAmount);
     }
 
+    /**
+     * @notice Withdraw from V1 vault to V1 vault
+     * @notice Waive fee if registered in vault registry
+     * @param share is the number of vault shares to be burned
+     * @param vault is the address of destination V1 vault
+     */
+    function withdrawToV1Vault(uint256 share, address vault)
+        external
+        nonReentrant
+    {
+        require(address(IRegistry) != address(0), "!IRegistry");
+        require(vault != address(0), "!vault");
+        require(share != 0, "!share");
+
+        bool feeless = IRegistry.canWithdrawForFree(address(this), vault);
```

Comment:
Should revert if this is false imo, if they want a withdrawal with fee, they should just call `withdraw`